### PR TITLE
Bump cardano-transactions

### PR DESCRIPTION
**Pull request by Anviking**
# Issue Number

Release.

# Overview

- [x] Bump cardano-transactions

# Comments

- List of downstream PRs included in the commit messages

### CODE REVIEW

**Review by KtorZ — APPROVED** [review]
We don't need any bump of these in principle. But why not.

**Comment by Anviking** [comment]
> We don't need any bump of these in principle. But why not.

Their binaries are going to be included as release artefacts, so good to have the latest right?

**Comment by KtorZ** [comment]
> so good to have the latest right?

The latest **release** yes. Ideally, we only want to pin released versions of cardano-addresses, cardano-transactions or cardano-node. Cardano-transactions hasn't been released in a while but we're back to it and will release soon. For now, let's just bump to the latest rev. For `cardano-addresses`, the latest release is 2.0.0 and we don't want to bump to something more recent yet.

File: stack.yaml [hunk 1]
```diff
@@ -46,15 +46,15 @@ extra-deps:
 
 # cardano-addresses-2.0.0
 - git: https://github.com/input-output-hk/cardano-addresses
-  commit: 9a64226fd749513ad0ed2e43e663f7132c92faa7
+  commit: de12008ee74faeccff396f46d985cb1066328b68
```
Comment (KtorZ) [inline stack.yaml]: Nope, let's stick to the previous commit here which corresponds to the latest release.

Comment (Anviking) [reply stack.yaml]: Reverted.

**Pull request merged**